src/app/http.interceptor.ts

```diff
@@ -192,7 +192,7 @@ export class InterceptedHttp extends Http {
       options.headers = new Headers();
     }
     options.headers.append("Content-Type", "application/json");
-    options.headers.append("Access-Control-Allow-Origin", "*");
+    // options.headers.append("Access-Control-Allow-Origin", "*");
```

Comment:
Are we managing the Access-Control-Allow-Origin from the backend? or need to add allowed origins here?